docs/source/configure.rst

```diff
@@ -36,6 +43,25 @@ The following steps guide you through those choices.
        # mkdir data
        # mkdir log
 
+3. Accept or change the default database management system used by Quipucords. By default, SQLite is used. You can choose to use SQLite or PostgreSQL.
+
+   - If you choose to use SQLite, no further configuration is required when you run the Docker command to start the server.
+   - If you choose to use PostgreSQL, you would use the following option when you run the Docker command to start the server: ``"QPC_DBMS=postgres"``.
+      - Additionally, you must provide the following information about your preconfigured postgres database:
```

Comment:
This says must provide but some of these are optional.  Maybe the subsections below should be organized like it is in install_scripted.rst to call out optional/required at the beginning of the bullet.

```
- **QPC_DBMS_PORT**
    - (Optional) Specifies the database port for postgres. Defaults to ``5432``.
- **QPC_DBMS_DATABASE**
    - (Optional) Specifies the database name for postgres. Defaults to ``postgres``.
- **QPC_DBMS_USER**
    - (Optional) Specifies the database user for postgres. Defaults to ``postgres``.
- **QPC_DBMS_PASSWORD**
    - (Required if ``QPC_DBMS=postgres``) Specifies the database password for postgres.
- **QPC_DBMS_HOST**
    - (Required if ``QPC_DBMS=postgres``) Specifies the database host for postgres.
```